relay-event-schema/src/protocol/event.rs

```diff
@@ -448,6 +448,10 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub csp: Annotated<Csp>,
 
+    /// NEL (Network Error Logging) reports.
+    #[metastructure(omit_from_schema)] // we only document error events for now
+    pub nel: Annotated<NetworkReport>,
```

Comment:
What's the rationale of making this a top-level field rather than a context?

Separately: During normalization, many of the fields will be cloned into standard places in the event payload. Do we have to store the full original report here, or could we store just the remaining fields that are NEL-specific? That would also allow us to _move_ the other fields.